### PR TITLE
Update repo for website sync

### PR DESCRIPTION
**Pull request by robbymilo**
This PR:
- Updates the markdown files to be compatible with the grafana website build. 
- Adds a GitHub action that syncs changes in the `content/tutorials` directory to the same directory in the website repo.
- Adds a command that uses a docker image to locally build the website and live preview changes to tutorials (see updated readme).

Todo:
- [x] Add grafanabot personal access token `GH_BOT_ACCESS_TOKEN` as a secret in this repo

Also see https://github.com/grafana/website/pull/999.

### CODE REVIEW

File: content/tutorials/build-a-data-source-plugin.md [hunk 1]
```diff
@@ -96,15 +90,15 @@ export interface MyQuery extends DataQuery {
 
 Now that you've defined the query model you wish to support, the next step is to bind the model to a form. The `FormField` is a text field component from `grafana/ui` that lets you register a listener, which will be invoked whenever the form field value changes.
 
-**QueryEditor.tsx**
+**QueryEditor.js**
```
Comment (marcusolsson) [inline content/tutorials/build-a-data-source-plugin.md]:
```suggestion
**QueryEditor.tsx**
```

Even if we're using highlighting for js, the file is still Typescript.

File: content/tutorials/build-a-data-source-plugin.md [hunk 2]
```diff
@@ -117,27 +111,26 @@ If you want the query to run automatically after updating the query, you can add
...
+```js
 export interface MyDataSourceOptions extends DataSourceJsonData {
   path?: string;
 }
 ```
 
 Just like query editor, the form field in the config editor calls the registered listener whenever the value changes.
 
-**ConfigEditor.tsx**
+**ConfigEditor.js**
```
Comment (marcusolsson) [inline content/tutorials/build-a-data-source-plugin.md]:
```suggestion
**ConfigEditor.tsx**
```

Even if we're using highlighting for js, the file is still Typescript.

File: content/tutorials/build-a-panel-plugin-with-d3.md [hunk 3]
```diff
@@ -30,35 +29,34 @@ In this tutorial, you'll build a simple, but complete bar graph panel.
...
+{{% tutorials/step duration="1" title="Data-driven transformations" %}}
 
 [D3.js](https://d3js.org/) is a JavaScript library for manipulating documents based on data. It lets you transform arbitrary data into HTML, and is commonly used for creating visualizations.
 
 In fact, D3.js is already bundled with Grafana, and you can access it by importing the `d3` package.
 
 - Import the `select` function from `d3`:
 
-**SimplePanel.tsx**
+**SimplePanel.js**
```
Comment (marcusolsson) [inline content/tutorials/build-a-panel-plugin-with-d3.md]:
```suggestion
**SimplePanel.tsx**
```

Even if we're using highlighting for js, the file is still Typescript.

File: content/tutorials/build-a-panel-plugin-with-d3.md [hunk 4]
```diff
@@ -126,61 +124,61 @@ draw() {
...
-## Theme your panel
+{{% /tutorials/step %}}
+{{% tutorials/step title="Theme your panel" %}}
 
 To provide your users with a consistent look-and-feel, you'll want to use the same colors as the built-in panels.
 
 In this step, you'll learn how to use the colors from the current theme.
 
-- In `SimplePanel.tsx`, add a `GrafanaTheme` property to the `PanelProps`.
+- In `SimplePanel.js`, add a `GrafanaTheme` property to the `PanelProps`.
```
Comment (marcusolsson) [inline content/tutorials/build-a-panel-plugin-with-d3.md]:
```suggestion
- In `SimplePanel.tsx`, add a `GrafanaTheme` property to the `PanelProps`.
```

File: content/tutorials/build-a-panel-plugin.md [hunk 5]
```diff
@@ -30,43 +29,41 @@ In this tutorial, you'll build a simple, but complete panel that will visualize
...
+{{% tutorials/step title="Panel plugins" %}}
 
 Since Grafana 6.x, panels are [ReactJS components](https://reactjs.org/docs/components-and-props.html). The simplest panel consists of a single function, the `render` function, which returns the content of the panel.
 
 Prior to Grafana 6.0, plugins were written in [AngularJS](https://angular.io/). Even though we still support plugins written in AngularJS, we highly recommend that you write new plugins using ReactJS.
 
 ### The `render` function
 
-The `render` function in `SimplePanel.tsx` determines how Grafana displays the panel in a dashboard.
+The `render` function in `SimplePanel.js` determines how Grafana displays the panel in a dashboard.
```
Comment (marcusolsson) [inline content/tutorials/build-a-panel-plugin.md]:
```suggestion
The `render` function in `SimplePanel.tsx` determines how Grafana displays the panel in a dashboard.
```

File: content/tutorials/build-a-panel-plugin.md [hunk 6]
```diff
@@ -30,43 +29,41 @@ In this tutorial, you'll build a simple, but complete panel that will visualize
...
 
 ### Panel properties
 
 The [PanelProps](https://github.com/grafana/grafana/blob/747b546c260f9a448e2cb56319f796d0301f4bb9/packages/grafana-data/src/types/panel.ts#L27-L40) interface exposes runtime information about the panel, such as panel dimensions, and the current time range.
 
 You can access the panel properties through `this.props`, as seen in your plugin.
 
-**SimplePanel.tsx**
+**SimplePanel.js**
```
Comment (marcusolsson) [inline content/tutorials/build-a-panel-plugin.md]:
```suggestion
**SimplePanel.tsx**
```

File: content/tutorials/build-a-panel-plugin.md [hunk 7]
```diff
@@ -87,41 +84,42 @@ Now that you can view your panel, make a change to the panel plugin:
...
+```js
 export const defaults: SimpleOptions = {
   text: 'The default text!',
 };
 ```
 
 Just like the panel itself, the panel editor is a React component, which returns a form that lets users update the value of the options defined by `SimpleOptions`.
 
-**SimpleEditor.tsx**
+**SimpleEditor.js**
```
Comment (marcusolsson) [inline content/tutorials/build-a-panel-plugin.md]:
```suggestion
**SimpleEditor.tsx**
```

File: README.md [hunk 8]
```diff
@@ -2,34 +2,25 @@
 
 [![License](https://img.shields.io/github/license/grafana/grafana)](LICENSE)
 
-[Grafana Tutorials](https://storage.googleapis.com/grafana-tutorials/index.html) are step-by-step guides that help you make the most of Grafana.
+[Grafana Tutorials](https://robbymilo-tutorials--grafana.netlify.com/tutorials/) are step-by-step guides that help you make the most of Grafana.
```
Comment (cinaglia) [inline README.md]:
```suggestion
[Grafana Tutorials](https://grafana.com/tutorials/) are step-by-step guides that help you make the most of Grafana.
```

**Pull request merged**